### DISCUSSION TITLE
Where are requirements located?

### DISCUSSION THREAD

**Discussion by GlennBags** [saloonphp/saloon#319] [General]
Thought I recalled that I saw requirements listed on the https://docs.saloon.dev/ page, but now I cannot find them. 
Can someone point me to a page? E.g. what version of PHP is required for each of the versions of Saloon?

**Comment by jdreesen** [comment]
- https://github.com/saloonphp/saloon/blob/v3/composer.json#L22
- https://github.com/saloonphp/saloon/blob/v3/composer.json#L22
- https://github.com/saloonphp/saloon/blob/v1/composer.json#L22

**Reply by GlennBags** [reply]
Appreciate it.

**Reply by Sammyjo20** [reply]
Thanks @jdreesen I will update the docs.
